mp/src/materialsystem/stdshaders/pbr_vs30.fxc

```diff
@@ -29,6 +30,14 @@ struct VS_INPUT
 	float2 vTexCoord0				: TEXCOORD0;
 	float4 vLightmapTexCoord		: TEXCOORD1;
 	float4 vLightmapTexCoordOffset	: TEXCOORD2;
+
+#if PARALLAXOCCLUSION
+//for parallax mapping..
+	float4 positionWS  				: POSITION;
+	float3 vNormalWS   				: NORMAL;
+	float3 vBinormalWS 				: BINORMAL;
```

Comment:
The pbr shader uses a different normal basis than source usually does. Do we need these?

Reply:
we need that to calculate the local position. thats how i did it by getting the data from the engine itself, I made it that way so that we can have parallax occlusion on 2.0b too. (2.0b removed the compute_tangent_frame function)

Reply:
Can you use `compute_tangent_frame` while on 3.0 instead of the source ones?

Reply:
`compute_tangent_frame` happens after the `parallaxCorrect` function, and takes the modified texCoords as input. im not sure if i can do it before.